### ISSUE TITLE
No indentation error with object destructuring assignment.

### ISSUE THREAD

**Issue by x1a0** [eslint/eslint#6813]
**What version of ESLint are you using?**
`v3.2.0`

**What parser (default, Babel-ESLint, etc.) are you using?**
`default`

**Please show your full configuration:**

``` js
{
  "extends": "airbnb",
  "env": {
    "mocha": true
  },
  "globals": {
    "__DEVELOPMENT__": false,
    "__PRODUCTION__": false,
    "__SERVER__": false,
    "__CLIENT__": false,
    "webpackIsomorphicToos": false,
    "createMockStore": false,
    "check": false,
    "standardComponentTests": false,
    "shallowWithIntl": false,
    "mountWithIntl": false
  },
  "parserOptions": {
    "ecmaVersion": 6,
    "sourceType": "module",
    "ecmaFeatures": {
      "experimentalObjectRestSpread": true,
      "jsx": true
    }
  },
  "rules": {
    // indent case-statements within a switch
    "indent": [2, 2, {"SwitchCase": 1}],

    // doesn't take statics into consideration so disabled for now
    "react/sort-comp": 0,

    // ignore common one-letter-vars  ("_" should be used to ignore an argument)
    "id-length": [2, {"min": 2, "properties": "never", "exceptions": [
      "_",
      "e",
      "a",
      "b",
      "x",
      "y"
    ]}],

    // support Immutable.js
    "new-cap": [2, {"newIsCap": true, "capIsNewExceptions": [
      "List",
      "OrderedSet",
      "Range",
      "Map",
      "Set"
    ]}]
  }
}
```

**What did you do? Please include the actual source code causing the issue.**
Source code:

``` js
import {
    a,
    b,
} from './ab';
```

**What did you expect to happen?**
I expect it gives error about incorrect indentation (should be 2 spaces however I use 4 spaces above).

**What actually happened? Please include the actual, raw output from ESLint.**
No error about indentation.

**Comment by gyandeeps** [comment]
Currently, the rule doesnt check for `import` or `export` specifiers.

**Comment by dbashford** [comment]
This isn't just `import`s as the title indicates but the example does not.  See screenshot below from demo page.

Note that it complains that I spaced the object properties in too far, but it doesn't complain that I spaced the `const` in too far.

![image](https://cloud.githubusercontent.com/assets/581127/18799501/e69b486c-81a5-11e6-9343-aaf95b31454a.png)
